SingularityUI/app/components/logs/FileNotFound.jsx

```diff
@@ -1,8 +1,36 @@
-import React from 'react';
+import React, {PropTypes} from 'react';
+
+const RENAMED_FILES = {
+  'service.log': 'tail_of_finished_service.log'
```

Comment:
Interesting. That's misconfigured locally, but it is correct in the deployed environments.